laborator/content/toolchain-decompilare/demo/compiling_steps/hello-world.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
+
+#include <stdio.h>
+#define LEN 200
+
+int main() {
+
+//    char v[LEN];
```

Comment:
What is this suposed to do?

Reply:
This is an example to see that after preprocessing that LEN will be replaced with 200. Right now it's commented because it's not used and the compiler will thrown an warning.